erpnext/agriculture/utils.py

```diff
@@ -16,5 +16,5 @@ def create_tasks(tasks, project_name, start_date):
 			"doctype": "Task",
 			"subject": task.get("task_name"),
 			"priority": task.get("priority"),
-			"project": project_name
+			"default_project": project_name
```

Comment:
This should add the project to the child table and mark it as default

Reply:
It's going to be the other way round. This PR makes the project table mandatory - https://github.com/Bloomstack/erpnext/pull/1656

Reply:
the default_project is read-only